cli/src/pcluster/models/param.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+# with the License. A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+# OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# This module contains all the classes representing the Resources objects.
+# These objects are obtained from the configuration file through a conversion based on the Schema classes.
+#
+
+
+class Param:
+    """Custom class to wrap a value and add more attributes."""
+
+    def __init__(self, value, default=None, valid: bool = True):
```

Comment:
here we could set `value=None` as default arg, so when we initialize it, we needn't pass None to the value in some cases. But just ignore this comment, since we will remove the Param class.